packages/manager/modules/emailpro/src/emailpro.controller.js

```diff
@@ -47,6 +47,10 @@ export default /* @ngInject */ function EmailProCtrl(
     $scope.changeOwnerUrl = link;
   });
 
+  const isRegionCA = function isRegionCA() {
+    return $scope.REGION === 'CA';
```

Comment:
`coreConfig` expose a `isRegion` method that could allow you to `coreConfig.isRegion('CA')`, maybe it could avoid having to declare this function